packages/gravity-ui-web/src/sass/05-components/03-organisms/_callout.scss

```diff
@@ -0,0 +1,44 @@
+.grav-c-callout {
+  position: relative;
+
+  p {
+    font-size: 1.5rem;
```

Comment:
For font sizes, please use one of the pre-defined ones we have - there's a scale from `-2` (smallest) to `6` (biggest), with `0` being the "base" size that's used as the default font size on the page. These are all responsive too, so they will increase relative to the viewport with, meaning you don't need that media query below to adjust the font size.

Looking at the designs, I think size `2` is a good fit - the card titles, which are slightly smaller, use `1` so I'm thinking one notch up from that is about right. :-)

To make things a bit easier, we have a mixin that sets the font-size using our scale, so you can replace this line (and the 2nd media query below) with this:
```sass
@include grav-font-size(2);
```